src/client/components/Story/Body.less

```diff
@@ -207,3 +207,9 @@
     }
   }
 }
+
+// This one is required to let rtl readers
+// to read both ltr and rtl (using .text-rtl) posts
```

Comment:
I don't see `.text-rtl` being used anywhere?

Reply:
It's a whitelisted class which can be used when composing a new post

```html
<div class="text-rtl">
This text will be aligned to the right to all busy users
</div>
```